pkg/controller/jenkinsimage/new.go

```diff
@@ -0,0 +1,67 @@
+package jenkinsimage
```

Comment:
can we move the contents of new.go to pkg/controller/jenkinsimage/resources/(individual resource).go

Reply:
Maybe also do this with the other new.go 
This is a common practice and is good for managing code.

Reply:
as discussed on slack, a next PR for this. Great